src/main/java/org/intellij/sdk/codesync/clients/ClientUtils.java

```diff
@@ -130,6 +130,25 @@ public static JSONResponse sendPost(String url, JSONObject payload, String acces
         }
     }
 
+    public static JSONResponse sendPost(String url, String accessToken) throws RequestError, InvalidJsonError {
```

Comment:
@syedgulahmed A POST request with no payload does not make much sense. We should be using a GET request here.

But if you must use a POST with empty data then you can remove code duplication here by

```
public static JSONResponse sendPost(String url, String accessToken) throws RequestError, InvalidJsonError {
    return sendPost(url, new JSONObject(), accessToken);
}
```